source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ACRChatWindow.mm

```diff
@@ -63,7 +63,7 @@ - (void)insertCard:(NSString *)card
     numberOfCards += 1;
 }
 
-- (void)insertView:(UIView *)view
+- (void)insertView:(ACRView *)view
```

Comment:
this change is ok, but I would like to know why changing to a more specific type of view